src/components/ui/button.tsx

```diff
@@ -55,4 +57,4 @@ const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
 );
 Button.displayName = "Button";
 
-export { Button };
+export { Button, buttonVariants };
```

Comment:
Just fyi, exporting `buttonVariants` triggers a linter warning..
```
/src/components/ui/button.tsx
  60:18  warning  Fast refresh only works when a file only exports components. Use a new file to share constants or functions between components  react-refresh/only-export-components
```

.. however, "Fast refresh" still works for button.

```suggestion
// eslint-disable-next-line react-refresh/only-export-components
export { Button, buttonVariants };
```